lms/envs/aws.py

```diff
@@ -123,6 +123,11 @@
 
 CELERYBEAT_SCHEDULE = {}  # For scheduling tasks, entries can be added to this dict
 
+############################# Persistent Grades ####################################
+
+# Queue to use for updating persistent grades
+RECALCULATE_GRADES_ROUTING_KEY = LOW_PRIORITY_QUEUE
```

Comment:
@efischer19 keep it tight to this for now, separate PR later to make review smaller/easier.